src/themes/default.css

```diff
@@ -75,6 +75,15 @@
   --bl-font-size-5xl: 3rem; /* 48px */
   --bl-font-size-6xl: 4rem; /* 64px */
 
+  /* Border radius */
+  --bl-border-radius-0: 0;
```

Comment:
I'm not sure if we should have a variable for `0` value since this means no border-radius. We don't have 0 values in size list. I think it's better to not include this too.